storage/StorageManager.cpp

```diff
@@ -556,10 +528,12 @@ MutableBlobReference StorageManager::getBlobInternal(const block_id blob,
     SpinSharedMutexSharedLock<false> read_lock(blocks_shared_mutex_);
     std::unordered_map<block_id, BlockHandle>::iterator it = blocks_.find(blob);
     if (it != blocks_.end()) {
-      DCHECK(it->second.block->isBlob());
+      DEBUG_ASSERT(it->second.block->isBlob());
       ret = MutableBlobReference(static_cast<StorageBlob*>(it->second.block), eviction_policy_.get());
     }
   }
+  // To be safe, release the shard for the blob after its read lock destructs.
```

Comment:
Same comment: Would it be clearer to replace "shard" with "lock held in the sharded hash table?"